### ISSUE TITLE
Upgrade to latest TinyMCE

### ISSUE THREAD

**Issue by stennie** [keystonejs/keystone#4699]
Keystone currently vendors TinyMCE 4.4.3 (2016-09-01) with some customisations. There have been a number of fixes and improvements since then: https://www.tinymce.com/docs/changelog/.

Check that uploadimage plugin (#4232, #4531, #4616) is also included.

**Comment by stennie** [comment]
@gautamsi Any open & unassigned issues with the [`help wanted`](https://github.com/keystonejs/keystone/labels/help%20wanted) label are up for grabs. No one is working on this yet, so thanks for volunteering!

There is a related suggestion to consider: move the bundled dependency to its own npm package: #2456. Do you want to look into feasibility of that approach?

Thanks,
Stennie

**Comment by gautamsi** [comment]
in my initial testing, I think it would be best if we remove the tinymce files from keystone repo and use it from npm package using require.resolve. We can also move the keystone skin to its own package and then properly make express route for keystone skin.

**Comment by gautamsi** [comment]
I am able to make this work with latest tinymce, moved keystone skin and uploadimage plugin to separate repo named `keystone-tinymce` (locally). this is achieved by adding following lines to `createStaticRouter.js`

```js
router.use('/js/lib/tinymce', express.static(path.dirname(require.resolve('tinymce'))));
var keystone_tinymce = path.dirname(require.resolve('keystone-tinymce'));
router.use('/js/lib/tinymce/skins/keystone', express.static(`${keystone_tinymce}/skin`));
router.use('/js/lib/tinymce/plugins/uploadimage', 
express.static(`${keystone_tinymce}/plugins/uploadimage`));
```

@stennie 
should I create my package as keystone-tinymce on npm or better you create this repo and let me push to that repo, and publish npm package.

**Comment by icdindia** [comment]
a few of the icons that belong to the new TinyMCE are not working in the Keystone backend. e.g. redo undo buttons.

Any hotfix for this at the moment?

**Comment by gautamsi** [comment]
Closed by #4897
